README.md

```diff
@@ -10,22 +10,23 @@ Adding shields would also be amazing -->
 
 ## Overview
 
-The [Dolby.io Real-time Streaming](https://dolby.io/products/real-time-streaming/) UIKit for iOS is designed to help iOS developers reduce the complexity of building a Dolby.io Real-time Streaming (RTS hereafter) monitor applications for iOS.
+The [Dolby.io Real-time Streaming](https://dolby.io/products/real-time-streaming/) UIKit for iOS is designed to help iOS developers reduce the complexity of building a Dolby.io Real-time Streaming (RTS) monitor applications for iOS.
 
 This package consists of three kinds of components:  
 
 * `DolbyIORTSUIKit`: The high-level UI components that can be used to develop a real-time streaming monitoring app for iOS with Dolby.io.  
 * `DolbyIORTSCore`: The logic between `DolbyIORTSUIKit` and Dolby.io [Real-time Streaming iOS SDK](https://docs.dolby.io/streaming-apis/docs/ios).
 * `DolbyIOUIKit`: The basic UI components used by `DolbyIORTSUIKit`.  
 
-> **_Info:_** There are two parties in RTS - a publisher and a viewer. A publisher is one who broadcasts the stream. A viewer(monitor) who consumes the stream. This UIKit is meant for viewer/monitor applications. Please refer to this [blog post](https://dolby.io/blog/real-time-streaming-with-dolby-io/) to understand the ecosystem.
+> **_Note:_** There are two parties in RTS - a publisher and a viewer. A publisher is one who broadcasts the stream. A viewer is one who consumes the stream. This UIKit is meant for viewer/monitor applications. Please refer to this [blog post](https://dolby.io/blog/real-time-streaming-with-dolby-io/) to understand the ecosystem.
 
 ## Requirements
 
 This setup guide is validated on both Intel and Apple Silicon based MacBook Pro machines running macOS 13.4.
 
 * Xcode Version 14.3.1 (14E300c)
 * iPhone device or simulator running iOS 15.0
+* Knowledges on iOS, Swift and SwiftUI programming
```

Comment:
Replace with :
` Familiarity with iOS development using Swift and Swift UI`

Reply:
Update on PR, https://github.com/DolbyIO/rts-uikit-ios/pull/21